server/etcdserver/api/etcdhttp/health_test.go

```diff
@@ -149,10 +140,13 @@ func TestHealthHandler(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			mux := http.NewServeMux()
+			lg := zaptest.NewLogger(t)
+			be, _ := betesting.NewDefaultTmpBackend(t)
```

Comment:
suggest to add `defer betesting.Close(t, be)`

Reply:
Okay. I will apply the suggested change. 

`NewDefaultTmpBackend` should auto close it after test is completed though.